README.md

```diff
@@ -85,9 +85,28 @@ Now the application will use the values defined in `.env`.
+
+| Variable          | Default                 | Description                                                                                                                                   |
+|-------------------|-------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|
+| PGDATABASE        | mobile_security_service | The database to connect to                                                                                                                    |
+| PGUSER            | postgresql              | The database user                                                                                                                             |
+| PGPASSWORD        | postgres                | The database password                                                                                                                         |
+| PGHOST            | localhost               | The database hostname to connect to                                                                                                           |
+| PGPORT            | 5432                    | The database port to connect to                                                                                                               |
+| PGSSLMODE         | disable                 | The SSL mode                                                                                                                                  |
+| PGCONNECT_TIMEOUT | 5                       | The default connection timeout (seconds)                                                                                                      |
+| PGAPPNAME         |                         | The [application_name](https://www.postgresql.org/docs/current/static/libpq-connect.html#LIBPQ-CONNECT-APPLICATION-NAME) connection parameter |
+| PGSSLCERT         |                         | The [sslcert](https://www.postgresql.org/docs/current/static/libpq-connect.html#LIBPQ-CONNECT-SSLCERT) connection parameter.                  |
+| PGSSLKEY          |                         | The [sslkey](https://www.postgresql.org/docs/current/static/libpq-connect.html#LIBPQ-CONNECT-SSLKEY) connection parameter.                    |
+| PGSSLROOTCERT     |                         | The [sslrootcert](https://www.postgresql.org/docs/current/static/libpq-connect.html#LIBPQ-CONNECT-SSLROOTCERT) connection parameter           |
+
 #### Database Entity Relationship Diagram
+
 <img src="https://user-images.githubusercontent.com/1596014/52293332-7229e380-296e-11e9-9bc4-5322f0fbe0ab.png" width="700" height="300">
 
 *Lucid Chart Link:* https://www.lucidchart.com/documents/edit/6293d791-8f70-47bb-8136-438128c27129/0
```

Comment:
Better we merge this one : https://github.com/aerogear/mobile-security-service then update this point here

Reply:
I think you're Adoc PR will be merged before this so I will probably have to pull in those changes and modify this before merging anyway :smile: